dev/io.openliberty.microprofile.telemetry.1.1.internal/src/io/openliberty/microprofile/telemetry/internal/cdi/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+/**
+ * @version 1.0
+ */
+@TraceOptions(traceGroup = "TELEMETRY", messageBundle = "io.openliberty.microprofile.telemetry.common.resources.MPTelemetry")
+@org.osgi.annotation.versioning.Version("1.0")
+package io.openliberty.microprofile.telemetry.internal.cdi;
```

Comment:
This `@Version` annotation still needs to be removed.

Reply:
Wait, I've confused myself slightly.

There are two versions of this file (one for mpTelemetry 1.0 and one for 1.1). Both should have the `@Version` removed because it's not exported.

Similarly there are two version of the rest package as well which need `@Version` removed because it's set in the bnd.bnd (or we could remove `@Version` from the bnd.bnd and set it in the package-info instead).